test/colony.js

```diff
@@ -189,25 +255,30 @@ contract("Colony", addresses => {
     });
 
     it("should fail if a non-admin user tries to make a task", async () => {
-      await checkErrorRevert(colony.makeTask(SPECIFICATION_HASH, 1, { from: OTHER }));
+      await checkErrorRevert(
```

Comment:
This appears to be a bit of a fast-and-lose-use of `checkErrorRevert`, both here and elsewhere in the PR. It seems to work, but I don't think the test would fail cleanly if the `makeTask` call succeeded; `makeTask` returns a `taskId`, which couldn't be used by `checkErrorRevert` to get a transaction receipt to check the status field...

(I actually don't think `checkErrorRevert` fails cleanly as-is if it's passed a promise for a transaction that succeeds, but this sort of usage would make it a lot harder)

Reply:
Ok, reverting this part will probably fix this.